tools/lint-md/lint-md.mjs

```diff
@@ -6,21 +6,20 @@ import remarkStringify from 'remark-stringify';
 import presetLintNode from 'remark-preset-lint-node';
 import { read } from 'to-vfile';
 import { reporter } from 'vfile-reporter';
+import { parseArgs } from 'util';
```

Comment:
Can you move it to the top of the file, so built-in modules are grouped together? And/or we could use the `node:` prefix